agents/modules_meshcore/amt-scanner.js

```diff
@@ -83,19 +83,23 @@ function AMTScanner() {
         return ((num >> 24) & 0xFF) + '.' + ((num >> 16) & 0xFF) + '.' + ((num >> 8) & 0xFF) + '.' + (num & 0xFF);
     }
 
-    this.scan = function (rangestr, timeout) {
+    this.scan = function (rangestr, timeout, callback) {
         var iprange = this.parseIPv4Range(rangestr);
         var rmcp = this.buildRmcpPing(0);
         var server = this.dgram.createSocket({ type: 'udp4' });
         server.parent = this;
         server.scanResults = [];
         server.on('error', function (err) { console.log('Error:' + err); });
         server.on('message', function (msg, rinfo) { if (rinfo.size > 4) { this.parent.parseRmcpPacket(this, msg, rinfo, function (s, res) { s.scanResults.push(res); }) }; });
-        server.on('listening', function () { for (var i = iprange.min; i <= iprange.max; i++) { server.send(rmcp, 623, server.parent.IPv4NumToStr(i)); } });
+        server.on('listening', function () { for (var i = iprange.min; i <= iprange.max; i++) {             
```

Comment:
Trailing whitespace.
```suggestion
        server.on('listening', function () { for (var i = iprange.min; i <= iprange.max; i++) {
```